solutions/bio1/MuTorere/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { SCodeBlock, SText, STitle, SCode, SSubtitle } from "components";
+
+export const MuTorere = () => {
+    return (
+        <>
+            <STitle>Idea</STitle>
+            <SText>
+                Like a lot of BIO1 Q2 problems, this problem involves
+                carefully reading the problem statement and implementing
+                exactly what is asked for. We can see that to solve the
+                $n$ command, we can just make a move exactly as
+                described in the problem statement, but it is not
+                immediately clear how to solve the $r$ command.
+            </SText>
+            <STitle>Solution</STitle>
+            <SSubtitle>Dealing with the $r$ command</SSubtitle>
+            <SText>
+                Notice that the game only has $9$ circles - the
+                "putahi" and the $8$ "kawai". Each circle can be in one of $3$ states:
```

Comment:
putahi and kawai better in italics i think

Reply:
How do you do italics?

Reply:
<i>text</i>
```
<i>text</i>
```